Templates/TimerTrigger-Powershell/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "defaultFunctionName": "TimerTriggerPowershell",
+    "description": "A Powershell function that will be run on a specified schedule",
+    "name": "TimerTrigger - Powershell",
+    "language": "Powershell",
+    "category": [
+        "Experimental",
+        "Data Processing"
```

Comment:
I'd say just leave this in Experimental like all the others. It's not our intent at this time to promote the Powershell templates. If we did, we'd want to do it for the other Powershell templates as well.